modules/Chromecast/resources/sender/mw.EmbedPlayerChromecast.js

```diff
@@ -571,8 +560,9 @@
         },
 
         getReceiverConfig: function () {
-            var chromecastConfig = this.getKalturaConfig( 'chromecast' );
-            return chromecastConfig.receiverConfig;
+            var receiverConfig = this.getKalturaConfig('chromecast').receiverConfig || {};
+            receiverConfig.defaultLanguageKey = this.beforeCastParams.captions[0] ? this.beforeCastParams.captions[0].lang : null;
```

Comment:
Don't you need null/undefined protection? you did it before in `activeBeforeCastCaptions`